plans/example/drop.go

```
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"github.com/testground/sdk-go/network"
+	"github.com/testground/sdk-go/runtime"
+	"github.com/testground/sdk-go/sync"
+)
+
+func ExampleDropNetwork(runenv *runtime.RunEnv) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 300*time.Second)
+	defer cancel()
+
+	runenv.RecordMessage("before sync.MustBoundClient")
+	client := sync.MustBoundClient(ctx, runenv)
+	defer client.Close()
+
+	if !runenv.TestSidecar {
+		return nil
```

This should return an error, the plan doesn't make sense without sidecar.